Workbooks/Virtual Machines - Performance Analysis/Performance Counters/Performance Counters.workbook

```diff
@@ -0,0 +1,1261 @@
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 172800000,
+                  "createdTime": "2018-10-04T22:01:18.374Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 259200000,
+                  "createdTime": "2018-10-04T22:01:18.375Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 604800000,
+                  "createdTime": "2018-10-04T22:01:18.375Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 1209600000,
+                  "createdTime": "2018-10-04T22:01:18.375Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 2592000000,
+                  "createdTime": "2018-10-04T22:01:18.375Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 5184000000,
+                  "createdTime": "2018-10-04T22:01:18.375Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 7776000000,
+                  "createdTime": "2018-10-04T22:01:18.375Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                }
+              ],
+              "allowCustom": true
+            }
+          },
+          {
+            "id": "112823d7-bf58-4604-a299-78ccc5899516",
+            "version": "KqlParameterItem/1.0",
+            "name": "LineChartQuerySnippet",
+            "type": 1,
+            "isRequired": false,
+            "query": "print(\"bin(TimeGenerated, {TimeRange:grain}), ResourceName | render linechart\")",
+            "crossComponentResources": [
+              "{Workspaces}"
+            ],
+            "isHiddenWhenLocked": true,
+            "timeContextFromParameter": null,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "7055a395-0d53-4724-93ad-15e03d4e2ffa",
+            "version": "KqlParameterItem/1.0",
+            "name": "GridQuerySnippet",
+            "type": 1,
+            "isRequired": false,
+            "query": "print(\"ResourceName\")",
```

Comment:
This hidden parameter doesn't seem necessary to me if all you're doing is printing the `ResourceName` string. Perhaps this is a relic of an earlier iteration?

Reply:
I see your point. I added it for completion - since I was doing "LineChartQuerySnippet". 
But I can remove it, unless we need to refer any other parameters.